api/src/models/api/stardust/IBlockResponse.ts

```diff
@@ -0,0 +1,10 @@
+import { IBlock } from "@iota/iota.js-stardust";
+import { IResponse } from "../IResponse";
+
+export interface IBlockResponse extends IResponse {
+    /**
+     * The deserialized block.
+     */
+    block?: IBlock;
+}
+
```

Comment:
It's just one empty line. I always add it at the end of every file to prevent github changes diff complain with "no newline at the end of file", and also seams to be a good practice.